test/foundry/new/helpers/FuzzMutationSelectorLib.sol

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import { FuzzTestContext } from "./FuzzTestContextLib.sol";
+import { FuzzMutations, MutationFilters } from "./FuzzMutations.sol";
+import { FuzzEngineLib } from "./FuzzEngineLib.sol";
+
+import { LibPRNG } from "solady/src/utils/LibPRNG.sol";
+
+import {
+    SignatureVerificationErrors
+} from "../../../../contracts/interfaces/SignatureVerificationErrors.sol";
+
+import { Vm } from "forge-std/Vm.sol";
+
+enum Failure {
+    InvalidSignature, // EOA signature is incorrect length
+    // InvalidSigner_BadSignature, // EOA signature has been tampered with
+    // InvalidSigner_ModifiedOrder, // Order with no-code offerer has been tampered with
+    // BadSignatureV, // EOA signature has bad v value
+    // BadContractSignature_BadSignature, // 1271 call to offerer, signature tampered with
+    // BadContractSignature_ModifiedOrder, // Order with offerer with code tampered with
+    // BadContractSignature_MissingMagic, // 1271 call to offerer, no magic value returned
+    // ConsiderationLengthNotEqualToTotalOriginal, // Tips on contract order or validate
+    // BadFraction_PartialContractOrder, // Contract order w/ numerator & denominator != 1
+    // BadFraction_NoFill, // Order where numerator = 0
+    // BadFraction_Overfill, // Order where numerator > denominator
+    length // NOT A FAILURE; used to get the number of failures in the enum
+}
```

Comment:
I'm still reading through this for the first time, so not sure it's even relevant, just wanted to make sure that `ErrorSpaceEnums.sol` was on your radar